docs/self-managed/platform-deployment/ingress-setup.md

```diff
@@ -0,0 +1,164 @@
+      optimize:
+        redirectUrl: "https://camunda.example.com/optimize"
+
+identity:
+  contextPath: "/identity"
+  fullURL: "https://camunda.example.com/identity"
+
+operate:
+  contextPath: "/operate"
+
+optimize:
+  contextPath: "/optimize"
+
+tasklist:
+  contextPath: "/tasklist"
+
+zeebe-gateway:
+  ingress:
+    enabled: true
+    className: nginx
+    host: "zeebe.camunda.example.com"
+```
+
+Using the custom values file, [deploy Camunda Platform as usual](../deployment.md):
+
+```shell
+helm install demo camunda/camunda-platform -f values-combined-ingress.yaml
+```
+
+Once deployed, you can access the Camunda Platform components on:
+
+- **Web applications:** `https://camunda.example.com/[identity|operate|optimize|tasklist]`
+- **Keycloak authentication:** `https://camunda.example.com/auth`
+- **Zeebe Gateway:** `grpc://zeebe.camunda.example.com`
+
+## Separated Ingress setup
+
+In this setup, each Camunda Platform component has its own Ingress/domain. There is no need to set the context (unless there is a good reason for that) since `/` is used as a default base. Here, we just need to set the Ingress configuration and authentication redirect URLs.
+
+```yaml
+# Chart values for the Camunda Platform 8 Helm chart in combined Ingress setup.
+
+# This file deliberately contains only the values that differ from the defaults.
+# For changes and documentation, use your favorite diff tool to compare it with:
+# https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform
+
+# IMPORTANT: Make sure to change "camunda.example.com" to your domain.
+
+global:
+  identity:
+    auth:
+      publicIssuerUrl: "https://keycloak.camunda.example.com/auth/realms/camunda-platform"
+      operate:
+        redirectUrl: "https://operate.camunda.example.com"
+      tasklist:
+        redirectUrl: "https://tasklist.camunda.example.com"
+      optimize:
+        redirectUrl: "https://optimize.camunda.example.com"
+
+identity:
+  ingress:
+    enabled: true
+    className: nginx
+    host: "identity.camunda.example.com"
+  fullURL: "https://identity.camunda.example.com"
+
+  keycloak:
+    ingress:
+      enabled: true
+      ingressClassName: nginx
+      hostname: "keycloak.camunda.example.com"
+    extraEnvVars:
+      - name: KEYCLOAK_PROXY_ADDRESS_FORWARDING
+        value: "true"
+      - name: KEYCLOAK_FRONTEND_URL
+        value: "https://keycloak.camunda.example.com"
+
+operate:
+  ingress:
+    enabled: true
+    className: nginx
+    host: "operate.camunda.example.com"
+
+optimize:
+  ingress:
+    enabled: true
+    className: nginx
+    host: "optimize.camunda.example.com"
+
+tasklist:
+  ingress:
+    enabled: true
+    className: nginx
+    host: "tasklist.camunda.example.com"
+
+zeebe-gateway:
+  ingress:
+    enabled: true
+    className: nginx
+    host: "zeebe.camunda.example.com"
+```
+
+Using the custom values file, [deploy Camunda Platform as usual](../deployment.md):
```

Comment:
This link is failing -- do you mean `platform-8-deployment.md`?

Reply:
this guide uses the new structure from #1241
I will move it there once the review is done